docs/hub/_toctree.yml

```diff
@@ -29,6 +29,10 @@
       title: Carbon Emissions
     - local: models-gated
       title: Gated models
+    - title: Model Card Guidebook
```

Comment:
This will need to be fixed before merged (see https://moon-ci-docs.huggingface.co/docs/hub/pr_545/en/model-card-annotated as the navbar does not work great rn) cc @Wauplin as you changed this part

Reply:
I have added an empty "Model Card Guidebook" page so that at least it shows in the left bar. But I am not sure docs can handle 3 levels of nesting (cc @mishig25 on that ?) (see picture below).

So maybe since the Model Cards section is completely getting revamped it would make sense to somehow include the "Carbon Emissions" and "Gated Repos" pages into the Guidebook and squeeze 1 level of nesting (e.g. the "Guidebook" will be the entire "Models > Model Cards" section instead of "Models > Model Cards > Model Card Guidebook")

WDYT ?

![2022-12-12_08-58](https://user-images.githubusercontent.com/11801849/206991908-faa81a12-dc24-4fb1-a9d0-cbc8e815a384.png)

Reply:
I think that would be great, personally. It's confusing to have these things at different levels, but also didn't want to step on toes or perhaps make something less visible that should be more visible.